images/go-ipfs/tests/docker-test.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+CONTAINER_NAME="ipfs-host-$(uuidgen)"
+CONTAINER_PORT=${FREE_PORT}
+REQUEST_RETRIES=5
+RETRY_DELAY=15
+
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
```suggestion
```

All looking good, just remove this since it isn't used anywhere and I think you're all set!